buildSrc/src/main/kotlin/tasks/InstallAndroidSdkTask.kt

```diff
@@ -0,0 +1,96 @@
+package tasks
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.GradleException
+import org.gradle.api.file.DirectoryProperty
+import org.gradle.api.file.ProjectLayout
+import org.gradle.api.provider.Property
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.Optional
+import org.gradle.api.tasks.OutputDirectory
+import org.gradle.api.tasks.TaskAction
+import org.gradle.tooling.GradleConnector
+import java.io.File
+import javax.inject.Inject
+
+/**
+ * Installs Android SDK. This task requires the ANDROID_SDK_ROOT env variable to be set.
+ */
+abstract class InstallAndroidSdkTask : DefaultTask() {
```

Comment:
I think this task should be untracked, since we don't really want to do up-to-date checks for it, right? Especially the files in the output directory would probably change when we run other builds on it.